src/CrudTrait.php

```diff
@@ -16,8 +16,11 @@
 
     public static function getPossibleEnumValues($field_name)
     {
+        $default_connection = Config::get('database.default');
+        $table_prefix = Config::get('database.connections.'. $default_connection .'.prefix');
+
         $instance = new static(); // create an instance of the model to be able to get the table name
-        $type = DB::select(DB::raw('SHOW COLUMNS FROM `'.Config::get('database.connections.'.env('DB_CONNECTION').'.prefix').$instance->getTable().'` WHERE Field = "'.$field_name.'"'))[0]->Type;
+        $type = DB::select(DB::raw('SHOW COLUMNS FROM `'.$table_prefix.$instance->getTable().'` WHERE Field = "'.$field_name.'"'))[0]->Type;
```

Comment:
This is confusing principally because of the **.** and **->** operators (i.e. is the *dot* operator a string concatenation vs an object method/property resolver).

PLUS what the raw SQL is entirely not escaped...